packages/core/src/logs/exports.ts

```diff
@@ -128,22 +136,26 @@ export function _INTERNAL_captureLog(
 
   const { level, message, attributes = {}, severityNumber } = log;
 
-  const serializedLog: SerializedOtelLog = {
-    severityText: level,
-    body: {
-      stringValue: message,
-    },
-    attributes: Object.entries(attributes).map(([key, value]) => logAttributeToSerializedLogAttribute(key, value)),
-    timeUnixNano: `${new Date().getTime().toString()}000000`,
-    traceId: traceContext?.trace_id,
-    severityNumber: severityNumber ?? SEVERITY_TEXT_TO_SEVERITY_NUMBER[level],
+  const serializedLog: SerializedLog = {
+    timestamp: timestampInSeconds(),
+    level,
+    body: message,
+    trace_id: traceContext?.trace_id,
+    severity_number: severityNumber ?? SEVERITY_TEXT_TO_SEVERITY_NUMBER[level],
```

Comment:
q: are we doing any kind of validation on `level` and `severityNumber` if they match? is there some kind of precedence one has over the other?